webapp/app/plugins/pluginpanel.html

```diff
@@ -141,12 +141,21 @@
       <ul class="content-spacer">
         <li class="small">
           <label>Job:</label>
-          <a title="Filter jobs like these"
-             href="{{buildbotJobnameHref}}"
+          <a title="Filter by job data strings (or buildername)"
+             href="{{jobSearchStrHref}}"
              prevent-default-on-left-click
-             ng-click="filterByBuildername(buildbotJobname)"
```

Comment:
This isn't just for ``buildernames`` anymore.  Could also be fields from task cluster jobs.